runtime/goroutine.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 // GoSafely wraps a `go func()` with recover()
-func GoSafely(wg *sync.WaitGroup, ignoreRecover bool, handler func(), finalFunc func(r interface{})) {
+func GoSafely(wg *sync.WaitGroup, ignoreRecover bool, handler func(), catchFunc func(r interface{})) {
```

Comment:
this func only called when catch exception, seems like to `catch (e) {}` not `finally {}`